sample-app/src/main/java/com/kogitune/prelollipoptransition/ListViewActivity.java

```diff
@@ -17,53 +17,51 @@
 
 package com.kogitune.prelollipoptransition;
 
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.AdapterView;
 import android.widget.AdapterView.OnItemClickListener;
 import android.widget.BaseAdapter;
 import android.widget.ListView;
 
+import com.kogitune.activity_transition.ActivityTransition;
 import com.kogitune.activity_transition.ActivityTransitionLauncher;
+import com.kogitune.activity_transition.ExitActivityTransition;
 
 public class ListViewActivity extends AppCompatActivity {
```

Comment:
Please run instrumentation test.
Sorry for no test on ci (I want to fix it..)

You can do instrumentation test like this.
![image](https://cloud.githubusercontent.com/assets/1386930/20640277/b3ea4c16-b41c-11e6-9bed-8f8d288c110e.png)